package.json

```diff
@@ -50,6 +51,7 @@
     "figlet": "1.2.1",
     "git-directory-deploy": "1.5.1",
     "husky": "1.3.1",
+    "jest": "^23.6.0",
```

Comment:
fix the version, renovate will ask for this.

Reply:
thanks!